feat(github-invite): missing member detection docs (#7926)
getsentry/sentry-docs: src/docs/product/integrations/source-code-mgmt/github/index.mdx

![How to disable Sentry comments on GitHub pull requests](github-org-settings.png)

### Missing Member Detection

> If your organization has users committing to Github repositories linked to Sentry that are not members of your organization, Sentry will remind organization owners and managers to invite these users through a banner on the members list page.

Reviewer: ```suggestion
If there are users committing to Github repositories linked to Sentry and they're not members of your organization, Sentry detects them as missing members. Once a month, Sentry sends organization owners and managers an email reminding them to invite those users to join their org. Sentry also shows a banner to invite missing members in the **Settings > Members** page.
```

![Github missing members invite banner](github-missing-members-banner.png)

> The banner will show the 5 most active contributors. To see all of the missing members (up to 50), click **View All** on the banner. To snooze the banner, click the ellipsis > **Hide Missing Members**.

Reviewer: ```suggestion
The banner shows the 5 most active contributors that are not yet in the organization. To see all missing members (up to 50), click "View All" in the banner. To snooze the banner, click the ellipsis, then select "Hide Missing Members".
```

Reviewer: So usually you should bold UI elements, but for some reason our docs only bold page names and we put UI elements in quotations lol. Suggested these edits for consistency.

// TODO: modal image

> You can select multiple users and assign roles and teams before inviting them through the modal.

Reviewer: Maybe this would be more obvious in practice, but it's unclear to me how you'd select multiple users and what the flow would be like from the image. Do you need to click "Invite Members" to do this?

Author: there are checkboxes on the modal (see https://github.com/getsentry/sentry/pull/55003) and then there's an "Invite Members" at the bottom

> 

Author: there is also a nudge email that will be sent out once a month to org owners and managers that haven't disabled this feature. should i include it here?

Reviewer: Suggested putting this in the first sentence

> This feature will be automatically enabled for you once your GitHub integration has been set up. To disable it, go to **Settings > Integrations > GitHub** and toggle it off.

Reviewer: ```suggestion
By default, this feature is automatically enabled once your GitHub integration has been set up. To disable it, go to **Settings > Integrations > GitHub** and toggle it off.
```

![How to disable Sentry comments on GitHub pull requests](github-org-settings.png)

### Missing Member Detection

If there are users committing to Github repositories linked to Sentry and they're not members of your organization, Sentry detects them as missing members. Once a month, Sentry sends organization owners and managers an email reminding them to invite those users to join their org. Sentry also shows a banner to invite missing members in the **Settings > Members** page.

![Github missing members invite banner](github-missing-members-banner.png)

The banner shows the 5 most active contributors that are not yet in the organization. To see all missing members (up to 50), click "View All" in the banner. To snooze the banner, click the ellipsis, then select "Hide Missing Members".

> 
> ![Github missing members invite modal](github-missing-members-modal.png)

Reviewer: ```suggestion
```
Move image to after the paragraph

Clicking "View All" on the invite banner will display the invite modal. From here, you can select multiple users and assign roles and teams before inviting them.

> 

Reviewer: ```suggestion

![Github missing members invite modal](github-missing-members-modal.png)

```
Move image to this line so it's shown after it's described for consistency
